pages/clubInfoPage.js

```diff
@@ -0,0 +1,149 @@
+const photos = ['photo1.jpg', 'photo2.jpg', 'photo3.jpg', 'photo4.jpg', 'photo5.jpg'];
+export const getServerSideProps = async () => {
+    try {
+      await clientPromise
+      return {
+        props: { isConnected: true },
+      }
+    } catch (e) {
+      console.error(e)
+      return {
+        props: { isConnected: false },
+      }
+    }
+  }
+export default function Home({
+    isConnected,
+  }) {
+return (
+  <div className="container">
+      <Head>
+        <title>GT Club Explorer Club Info Page</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      {/* <NavBar />*/}
+      <Box
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        marginBottom="40px"
+      >
+        <Image
+          src="/img/loremIpsum.png"
+          width="100%"
+          height="100px"
+          objectFit="cover"
+        />
+      </Box>
+      <Container maxW='container.xl' mx="auto" spacing={16} marginBottom="40px">
+        <Flex alignItems="center">
+          <Avatar
+            size="2xl"
+            mx="auto"
+            display={{ base: "none", lg: "block" }}
+            borderRadius='full'
+            src="/img/LoremIpsumAboutPhoto.PNG"
+            marginRight="40px"
+          />
+          <Heading
+            fontSize={{ base: '5xl', lg: '7xl' }}
+            fontWeight='extrabold'
+          >
+            Georgia Tech Lorem Ipsum Association
+          </Heading>
+        </Flex>
+      </Container>
+      <Container maxW='container.lg' mx="auto" spacing={16}>
+        <Box backgroundColor= 'purple.100'
+          padding= '10px'
+          borderRadius= '20px'
+          display= 'inline-block'
+          margin= '5px'>
+          <Text textColor= 'purple'>Cultural</Text>
+        </Box>
+
+        <Box backgroundColor= 'blue.100'
+          padding= '10px'
+          borderRadius= '20px'
+          display= 'inline-block'
+          margin= '5px'>
+          <Text textColor= 'blue'>Entertainment</Text>
+        </Box>
+
+        <Box backgroundColor= 'red.100'
+          padding= '10px'
+          borderRadius= '20px'
+          display= 'inline-block'
+          margin= '5px'>
+          <Text textColor= 'red.600'>Career</Text>
+        </Box>
+        <Box backgroundColor= 'green.100'
+          padding= '10px'
+          borderRadius= '20px'
+          display= 'inline-block'
+          margin= '5px'>
+          <Text textColor= 'green'>Mechanical Engineering</Text>
+        </Box>
+        <Box backgroundColor= 'yellow.100'
+          padding= '10px'
+          borderRadius= '20px'
+          display= 'inline-block'
+          margin= '5px'>
+          <Text textColor= 'yellow.600'>Spahgetti Enthusiasts</Text>
+        </Box>
+      </Container>
+
+      <Container maxW='container.lg' mx="auto" spacing={16} marginBottom="20px" marginTop="20px">
```

Comment:
Starting from this section, let's use a markdown parser: https://www.npmjs.com/package/@uiw/react-md-editor.

The one linked above is pretty easy to use & their docs are really good

Reply:
fixed